sp800-63b/sec5_authenticators.md

```diff
@@ -348,3 +348,15 @@ Biometrics are also used in some cases to prevent repudiation of registration an
 Authenticators that are directly connected to or embedded in endpoints MAY convey attestation information such as the provenance or health and integrity of the authenticator (and possibly the endpoint as well) to the verifier as part of the authentication protocol. If this attestation is signed, the verifier SHOULD validate its signature. This information MAY be used as part of a risk-based authentication decision.
 
 When federated authentication is being performed as described in [SP 800-63C](sp800-63c.html), the verifier SHOULD include any such attestation information in the assertion it provides to the relying party.
+
+#### <a name="verifimpers"></a>5.2.5 Verifier impersonation resistance
+
+Verifier impersonation attacks, sometimes referred to as "phishing attacks", refer to attempts by fraudulent verifiers and relying parties to fool an unwary claimant into authenticating to an impostor website. In previous editions of the Guideline, protocols that are resistant to verifier impersonation attacks were also referred to as "strongly man-in-the-middle resistant".
```

Comment:
I'm probably being inconsistent in terms. I wasn't sure about "in previous versions of this document" because this is actually the first version of -63B. I thought that calling it "the Guideline" was more inclusive of the document suite, but 800-63-2 is probably easier to understand.